### PR TITLE
add

### PR DESCRIPTION
**Pull request by Sergii112**

### CODE REVIEW

File: tests/regForm_DZ18.spec.js [hunk 1]
```diff
@@ -0,0 +1,305 @@
+import { test, expect } from "@playwright/test";
+
+test.describe("Registration form", () => {
+    test.describe("Name field validation", () => {
+        test.beforeEach(async ({ page }) => {
+            await page.goto("https://qauto.forstudy.space/");
```
Comment (sttaran) [inline tests/regForm_DZ18.spec.js]: you baseUrl in you playwright config, use relative path

File: tests/regForm_DZ18.spec.js [hunk 2]
```diff
@@ -0,0 +1,305 @@
...
+        });
+
+        test("Positive test - valid name input", async ({ page }) => {
+            const nameInput = page.locator('#signupName');
+            const validName = "Will";
+
+            await nameInput.fill(validName);
+            await nameInput.press("Enter");
+
+            await expect(nameInput).not.toHaveClass("invalid");
```
Comment (sttaran) [inline tests/regForm_DZ18.spec.js:18]: use https://playwright.dev/docs/api/class-locatorassertions#locator-assertions-to-have-css

Comment (sttaran) [reply tests/regForm_DZ18.spec.js:18]: you should check border color in every negative scenario

File: tests/regForm_DZ18.spec.js [hunk 3]
```diff
@@ -0,0 +1,305 @@
...
+        });
+
+        test("Negative test - wrong length input", async ({ page }) => {
+            const lastNameInput = page.locator('#signupLastName');
+            const invalidName = "A";
+            const errorMessage = page.locator('.invalid-feedback');
+
+            await lastNameInput.fill(invalidName);
+            await lastNameInput.press("Enter");
+            await lastNameInput.focus()
```
Comment (sttaran) [inline tests/regForm_DZ18.spec.js]: these 2 rows probably are redundant in every test

File: tests/regForm_DZ18.spec.js [hunk 4]
```diff
@@ -0,0 +1,305 @@
...
+           await passwordInput.focus()
+           await passwordInput.blur()
+
+           await expect(errorMessage).toContainText("Password required");
+       });
+
+        test("Negative test - wrong password format input", async ({ page }) => {
+           const passwordInput = page.locator("#signupPassword");
+           const invalidPassword = "password";
+           const errorMessage = page.locator('.invalid-feedback');
```
Comment (sttaran) [inline tests/regForm_DZ18.spec.js]: `const errorMessage = passwordInput.locator(' + .invalid-feedback');`

**Review by sttaran — CHANGES_REQUESTED** [review]

**Review by Sergii112 — COMMENTED** [review]
done

**Pull request merged**